src/index.test.js

```diff
@@ -155,6 +157,23 @@ describe('API', () => {
 
   describe('Access token managament', () => {
     describe('when calling the api ', () => {
+      it('calls access token endpoint with creds', async () => {
+        const accessTokenEndpointCall = nock(US_BASE_ENDPOINT_URL, {
+            reqheaders: {
+              'Content-Type': 'application/x-www-form-urlencoded'
+            }
+          })
+          .post('/v1/token', { grant_type: 'client_credentials' })
```

Comment:
Que tal?
O mock consegue entender assim, se for enviado corretamente os params?

```suggestion
          .post('/v1/token? grant_type=client_credentials')
```

Caso não, e a [essa suggestão ](https://github.com/inloco/incognia-api-node-wrapper/pull/11/commits/ce06148b9d0d9f20a58b2858faddbdd9b0e32042#r649676097) funcione, acredito que fica assim

```suggestion
          .post('/v1/token', null, { params: { grant_type: 'client_credentials' } })
```